src/mscorlib/src/System/Threading/Tasks/future.cs

```diff
@@ -420,10 +404,6 @@ public Task(Func<object, TResult> function, object state, CancellationToken canc
             {
                 ThrowHelper.ThrowArgumentNullException(ExceptionArgument.scheduler);
             }
-            if ((internalOptions & InternalTaskOptions.SelfReplicating) != 0)
-            {
-                ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument.creationOptions, ExceptionResource.TaskT_ctor_SelfReplicating);
```

Comment:
Fixed.  I removed the ExceptionResource, but not the ExceptionArgument, as it's used elsewhere.